### ISSUE TITLE
As an admin, I want to be notified if a funder funds a bounty with an extremely low price because setting reasonable and fair hourly rates is one of many ways that OSS is sustained. 

### ISSUE THREAD

**Issue by frankchen07** [gitcoinco/web#2808]
### User Story

As an admin, I want to be notified if a funder funds a bounty with an extremely low price because setting reasonable and fair hourly rates is one of many ways that OSS is sustained. 

### Why Is this Needed

OSS sustainability is only possible if we pay developers a fair price for the value they put forth. Certain prices (e.g < $1) for any type of task is considered lowballing the developer / bounty hunter. Warning Gitcoin admins of an extremely low bounty price and giving the admins the option to manually follow up with the funder is necessary to prevent publishing of lowball bounties.

### Current Behavior

Currently, there are no warnings or emails triggered when a bounty is funded below a certain threshold.

### Expected Behavior

An email is automatically sent to Gitcoin administrators when a bounty is funded that is below $X. 

### Definition of Done

1. When a funder hits "submit an issue," a threshold check (if else logic) on the bounty value is performed.
2. If the bounty value is below the threshold, an automatic email to Gitcoin admins is triggered with the issue link, funder, and funder information.
3. Being below a $X threshold should not prevent the bounty from being published, but an automatic email should be triggered.

**Comment by gitcoinbot** [comment]
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.25 ETH (52.61 USD @ $210.44/ETH)  attached to it as part of the Gitcoin fund.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/2808/1753).
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/2808/1753).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $46,208.01 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by kuhnchris** [comment]
Additionally I may also want to point, that we could include a "The average hourly rate on the platform currently is 0.XXX ETH per hour" when creating a bounty, and alert the user if his price (per estimated h) is lower than the 'platform average'. (just as an idea)

**Comment by frankchen07** [comment]
@usmanmuhd - since you were the first, I've approved you for work! 

@kuhnchris - there are actually two tickets open to improve pricing:

https://github.com/gitcoinco/web/issues/2158
https://github.com/gitcoinco/web/issues/2117

**Comment by gitcoinbot** [comment]
@usmanmuhd Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=100)

**Comment by usmanmuhd** [comment]
Will be sending in a WIP PR by today

**Comment by gitcoinbot** [comment]
@usmanmuhd Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=100)

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. **Started** 3. Submitted 4. Done 

<hr>

@usmanmuhd due to inactivity, we have escalated [this issue](https://gitcoin.co/issue/gitcoinco/web/2808/1753) to Gitcoin's moderation team. Let us know if you believe this has been done in error!

* [x] reminder (3 days)
* [x] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=100)

**Comment by usmanmuhd** [comment]
Will be freeing up the bounty for some other bounty hunter. Keeping very busy with assignments and assessments

**Comment by kuhnchris** [comment]
Did you do any work or shall we create from scratch?

**Comment by usmanmuhd** [comment]
I did not do much except a very basic HTML page. So yeah, it should be from scratch.

**Comment by kuhnchris** [comment]
OK, thank you. Mind "Stop Work" on Github?

**Comment by usmanmuhd** [comment]
I did that already.

**Comment by gitcoinbot** [comment]
@frederikbolding Hello from Gitcoin Core - are you still working on this issue? Please submit a WIP PR or comment back within the next 3 days or you will be removed from this ticket and it will be returned to an ‘Open’ status. Please let us know if you have questions!
* [x] reminder (3 days)
* [ ] escalation to mods (6 days)

Funders only: Snooze warnings for [1 day](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=1) | [3 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=3) | [5 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=5) | [10 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=10) | [100 days](https://gitcoin.co/issue/gitcoinco/web/2808/1753?snooze=100)

**Comment by FrederikBolding** [comment]
Sorry, I wasn't aware that I had this issue. I've stopped work as I'm busy with another issue 😄

**Comment by oddballbutler** [comment]
So I'm thinking the change should go in dashboard.helpers.process_bounty_changes(). Did you have any idea about how the low bounty threshold value should be generated?

**Comment by oddballbutler** [comment]
Also, should I send the email to founders@gitcoin.co? Or something else?

**Comment by kuhnchris** [comment]
@PixelantDesign @frankchen07 any infos for oddballbutler?

**Comment by oddballbutler** [comment]
So I'm seeing that danlipert already has a pull request for this... I wasn't aware that someone else was working on this when I applied for the issue on gitcoin. Is there a way around this kind of problem?

**Comment by frankchen07** [comment]
I'd say the low bounty threshold can be hardcoded @ $10 for now, given if something is priced at that, the highest going hourly rate would be $10/hr, which is quite low.

**Comment by kuhnchris** [comment]
@frankchen07 that depends tho! what if the bounty was made at a time when ETH was at a all time high (300$+), a bounty with .3 eth = 30$, with today's price that'd be less than 8$. Would that notification run daily and consider the current prices or shall that be a one time run?

**Comment by frankchen07** [comment]
@kuhnchris - that's an excellent point. Thanks for bringing it up.

I'd trend towards not over-engineering this for now, and use the USD value _at the time the bounty was created._

Your point about the volatility of ETH is spot on, but I'm more concerned with lowballers at the onset of bounty creation, rather than normal bounties that "turn into" lowballs simply due to market volatility. The former is a condition of the funder, whereas the second, is the market.

**Comment by kuhnchris** [comment]
That's true. The first one can be actively influenced, the second one is whale-pending on the price.
So it makes sense to inform the user during creation/relatively early in the process about the low-balls price.

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. **Submitted** 4. Done 

<hr>

__Work for 0.25 ETH (21.1 USD @ $84.39/ETH) has been submitted by__:

 1. [@oddballbutler](https://gitcoin.co/profile/oddballbutler)



@frankchen07 please take a look at the submitted work:
* [PR](https://github.com/gitcoinco/web/pull/3219) by @oddballbutler

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/2808/1753)
* Want to chip in? Add your own contribution [here](https://gitcoin.co/issue/gitcoinco/web/2808/1753).
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $44,888.36 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by ryan-shea** [comment]
Hey @frankchen07, looks like this one is ready to be reviewed!

**Comment by frankchen07** [comment]
thanks - will try my best to build this into the next sprint, get it merged, and get @oddballbutler paid out. Thank for being patient, lots of moving pieces.

**Comment by oddballbutler** [comment]
@frankchen07 Just got an email saying that the gitcoin issue expires in two days? Was this ever taken care of? I would like to be paid for my work...

**Comment by PixelantDesign** [comment]
We will get this reviewed. Sorry for the delay.

**Comment by kuhnchris** [comment]
Hey there @oddballbutler - seems like the team is currently in event-travel mode. I added a couple of question and one review point in the PR, please check them and get back to us, if that's fine I think we can go ahead and merge that change in.

Thanks!
Chris

**Comment by owocki** [comment]
@danlipert @PixelantDesign can we pls get this reviewed?  thanks

**Comment by oddballbutler** [comment]
@kuhnchris ok thanks. I won't be able to work on it for a few weeks though.

**Comment by kuhnchris** [comment]
It depends on how fast @owocki and co needs it, if it's just regarding the settings move we can do it was well, if it's suddently urgent. else, we'd wait, i guess. :-)

**Comment by gitcoinbot** [comment]
⚡️ A tip worth 0.00750  ETH (1.41 USD @ $188.01/ETH) has been granted to @oddballbutler for this issue from @frankchen07. ⚡️ 

Nice work @oddballbutler! Your tip has automatically been deposited in the ETH address we have on file.

 * $130663.07 in Funded OSS Work Available at: https://gitcoin.co/explorer
 * Incentivize contributions to your repo: <a href='https://gitcoin.co/tip'>Send a Tip</a> or <a href='https://gitcoin.co/funding/new'>Fund a PR</a>
 * No Email? Get help on the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>

**Comment by gitcoinbot** [comment]
Issue Status: 1. Open 2. Started 3. Submitted 4. **Done** 

<hr>

__The funding of 0.25 ETH (47.0 USD @ $188.01/ETH)  attached to this issue has been approved & issued to @oddballbutler.__  

 

Additional Tips for this Bounty:
* frankchen07 tipped 0.0075 ETH worth 1.41 USD to oddballbutler.

<hr>

* Learn more [on the Gitcoin Issue Details page](https://gitcoin.co/issue/gitcoinco/web/2808/1753)
* Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
* $130,616.07 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)

**Comment by Sirlupinwatson1** [comment]
Congrats @frankchen07
